src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/DateTimeSearchParam.sql

```diff
@@ -8,7 +8,10 @@
     IsLongerThanADay bit NOT NULL,
     IsHistory bit NOT NULL,
     IsMin bit CONSTRAINT date_IsMin_Constraint DEFAULT 0 NOT NULL,
-    IsMax bit CONSTRAINT date_IsMax_Constraint DEFAULT 0 NOT NULL
+    IsMax bit CONSTRAINT date_IsMax_Constraint DEFAULT 0 NOT NULL,
+    CONSTRAINT PK_DateTimeSearchParam PRIMARY KEY NONCLUSTERED(ResourceTypeId, ResourceSurrogateId, SearchParamId, StartDateTime, EndDateTime)
+    WITH (DATA_COMPRESSION = PAGE) 
+	ON PartitionScheme_ResourceTypeId(ResourceTypeId)
```

Comment:
>PartitionScheme_ResourceTypeId [](http://example.com/codeflow?start=4&length=30)

Unrelated to content of your PR, but can you enlighten me  about purpose of this file?

We have currentschema.sql, is these files are purely to make reading easier?

Reply:
You are right. All the files under /Features/Schema/Sql are used to generate a full schema script file automatically. We don't need to add the currentschema.sql file anymore it gets auto-generated and we still continue to manually add diff.sql file